config/crds/machine.crd.yaml

```diff
@@ -83,6 +83,18 @@ spec:
               type: object
           type: object
   version: v1alpha1
+  additionalPrinterColumns:
+  - JSONPath: .spec.providerConfig.value.instanceType
```

Comment:
Will not work with machine classes

Reply:
> Will not work with machine classes

I'm going to close this PR and we can discuss offline how we can go about getting custom columns rendered.